assets/js/index.js

```diff
@@ -3,8 +3,16 @@
 
   function setupSelect2For(name) {
     const perPage = 10;
+    const input = $('select[name="' + name + '"]');
+    const form = input.parents('form');
 
-    $('select[name="' + name + '"]').select2({
+    function _submitFormIfPressedEnter(e) {
+      if (e.keyCode === 13) {
```

Comment:
I tried with it, but felt it made the "blinking" worse, as it adds another blink where the input becomes grey.

It's stupid that there's no easier way of allowing ENTER to submit the form, which means we have to live with these hacks and the flash of new autocomplete suggestions appearing between pressing ENTER and the form being submitted.